File: app/src/main/java/home/oleg/placesnearme/common/converter/DrawableConverter.java
package home.oleg.placesnearme.common.converter;

import android.graphics.drawable.Drawable;
import android.support.annotation.DrawableRes;

import com.google.android.gms.maps.model.BitmapDescriptor;

import io.reactivex.annotations.NonNull;

/**
 * Created by Oleg Sheliakin on 28.08.2018.
 * Contact me by email - olegsheliakin@gmail.com
 */
public interface DrawableConverter {

    BitmapDescriptor convert(@NonNull Drawable drawable);

}

File: feature-map/src/main/java/home/oleg/placesnearme/feature_map/drawable_converter/DrawableConverter.kt
package home.oleg.placesnearme.feature_map.drawable_converter

import android.graphics.drawable.Drawable

import com.google.android.gms.maps.model.BitmapDescriptor

import io.reactivex.annotations.NonNull

/**
 * Created by Oleg Sheliakin on 28.08.2018.
 * Contact me by email - olegsheliakin@gmail.com
 */
interface DrawableConverter {
    fun convert(drawable: Drawable): BitmapDescriptor
}
